brigade.js

```diff
@@ -178,6 +180,29 @@ function runSuite(e, p) {
   }
 }
 
+// handleIssueComment handles an issue_comment event, parsing the comment
+// text and determining whether or not to trigger a corresponding action
+function handleIssueComment(e, p) {
+  payload = JSON.parse(e.payload);
+
+  // Extract the comment body and trim whitespace
+  comment = payload.body.comment.body.trim();
+
+  // Here we determine if a comment should provoke an action
+  switch(comment) {
+    case "/test":
```

Comment:
It's totally up to us (or, in general, each `brigade.js` author) on what sort of comments we want to designate as official phrases/words to trigger actions.

In this example, we use `/test` to trigger the entire default test/build suite, but we could use a different phrase.

We might like to have cases for each particular check run _or_ we may decide that the one, do-all comment to kick off the entire suite is all we need (since, remember, once the suite is run, any failing checks can be re-run via the Check Suite UI on a given PR.)

Reply:
Since we also have `/azp run` to trigger the azure pipeline, can we call it `/brig run` to help people remember?

Reply:
Good call!  Updated.  Only have the `/brig run` handler for now... as you've mentioned, we can tweak/update after seeing how we use it day-to-day...